docker/cygnus-ngsi/Dockerfile

```diff
@@ -253,11 +253,6 @@ ENV CYGNUS_ARCGIS_BATCH_SIZE ""
 ENV CYGNUS_ARCGIS_BATCH_TIMEOUT ""
```

Comment:
We have some other 2 Dockerfiles in this repository. Not sure if they are quite used or mantained :), but maybe it worth the time to have a look to check if similar changes apply there...

Reply:
Not sure about this @fgalan, this solution works for automated builds (like dockerhub), and only cygnus-ngsi is build by this kind of jobs.

Anyway, it's not that complicated to make this changes on those images as well, so let me know if you thing it would be helpful to modify those images as well.

Reply:
Maybe it doesn't worth the effort if they images that could be unmantained, but al least to include a FIXME mark in their dockerfiles to refers this PR. Something like this:

```
# FIXME: this image could be obsolete. If re-activate, then probably the changes done in 
# PR https://github.com/telefonicaid/fiware-cygnus/pull/1801 should be also done in this Dockerfile
```

Reply:
Fixed in [8b12c48](https://github.com/telefonicaid/fiware-cygnus/pull/1801/commits/8b12c487bb6c266e9d362f31375ead5706608a46)